Source/CombatExtended/CombatExtended/WorldObjects/HostilitySheller.cs

```diff
@@ -193,12 +197,21 @@ private void LaunchProjectile(ThingDef projectileDef)
             shell.tileInt = comp.parent.Tile;
             shell.SpawnSetup();
             Find.World.worldObjects.Add(shell);
+            if (shooter == null && comp.parent.Faction.def.humanlikeFaction)
+            {
+                Log.Warning("CE: Shooter of HostilitySheller from " + comp.parent.ID + ", " + comp.parent.Faction.name + " is null, regenerating");
```

Comment:
Since we have a solution for this warning, I don't know that it's worth logging a warning here.  Let's save the warning for when we can't reconstruct the shooter.